verification/curator-service/ui/src/components/Profile.tsx

```diff
@@ -24,11 +42,310 @@ const styles = makeStyles((theme: Theme) => ({
+                                id="oldpassword"
+                                type={passwordVisible ? 'text' : 'password'}
+                                value={formik.values.oldpassword}
+                                onChange={formik.handleChange}
+                                endAdornment={
+                                    <InputAdornment position="end">
+                                        <IconButton
+                                            aria-label="toggle password visibility"
+                                            onClick={() =>
+                                                setPasswordVisible(
+                                                    !passwordVisible,
+                                                )
+                                            }
+                                            edge="end"
+                                        >
+                                            {passwordVisible ? (
+                                                <Visibility />
+                                            ) : (
+                                                <VisibilityOff />
+                                            )}
+                                        </IconButton>
+                                    </InputAdornment>
+                                }
+                                label="Old password"
+                            />
+                            <FormHelperText>
+                                {formik.touched.oldpassword &&
+                                    formik.errors.oldpassword}
+                            </FormHelperText>
+                        </FormControl>
+
+                        <FormControl
+                            disabled={disabled}
+                            className={classes.inpputField}
+                            variant="outlined"
+                            error={
+                                formik.touched.password &&
+                                Boolean(formik.errors.password)
+                            }
+                        >
+                            <InputLabel htmlFor="password">Password</InputLabel>
+                            <OutlinedInput
+                                fullWidth
+                                id="password"
+                                type={passwordVisible ? 'text' : 'password'}
+                                value={formik.values.password}
+                                onChange={formik.handleChange}
+                                endAdornment={
+                                    <InputAdornment position="end">
+                                        <IconButton
+                                            aria-label="toggle password visibility"
+                                            onClick={() =>
+                                                setPasswordVisible(
+                                                    !passwordVisible,
```

Comment:
This should have it's own variable in React state, right now after toggling `oldPassword` visibility `newPassword` visibility changes as well and the other way round. All password inputs should have separate visibility state variable.